modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -12,11 +12,9 @@ def initialize(user)
 
         def get_appointments(start_date:, end_date:, pagination_params:)
           response = vaos_v2_appointments_service.get_appointments(start_date, end_date, nil, pagination_params)
-          response[:data].each do |appt|
-            appt[:location_id] = Mobile::V0::Appointment.convert_from_non_prod_id!(appt[:location_id])
-          end
-          appointments = merge_clinics(response[:data])
-          appointments = merge_facilities(appointments)
+
+          appointments = merge_clinic_facility_address(response[:data])
```

Comment:
don't love this name but `merge_clinic_address` seems weird since if you dig into it, it calls a facility endpoint. at least this way, whoever reads this know that it's not a parent facility.